src/util.rs

```diff
@@ -24,17 +23,31 @@ pub struct DeviceHandle {
     pub queue: Queue,
 }
 
+#[derive(Error, Debug)]
+pub enum SurfaceCreationError {
+    #[error("Error creating device")]
+    CantCreateDevice,
+    #[error(transparent)]
+    WgpuSurface(#[from] wgpu::CreateSurfaceError),
+}
+
+impl Default for RenderContext {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 impl RenderContext {
-    pub fn new() -> Result<Self> {
+    pub fn new() -> Self {
```

Comment:
Once again, breaking change. Where did the error go?

Reply:
Here again, there was no way to return an `Err`, just panic or `Ok` at the end

Reply:
I don't think we currently have to be that careful with breaking changes, when they make sense. vello is still (pre-)alpha. (Especially if they create less pain, changing to from `Self` -> `Result<Self>` would be more pain to refactor downstream). 

That said, I haven't yet looked deep enough whether this can not fail under normal circumstances though.